modules/reserved-ips/README.md

```diff
@@ -0,0 +1,79 @@
+# IBM Reserved IP Module
+
+This module creates reserved ip's (https://cloud.ibm.com/docs/vpc?topic=vpc-managing-ip-addresses&interface=ui) which gives you the capability
+on VPC that allows you to reserve IP addresses for use on your resources.
```

Comment:
The module creates reserved IP's (https://cloud.ibm.com/docs/vpc?topic=vpc-managing-ip-addresses) on existing subnets. Reserved IP's can be assigned to your resources, for example VPE endpoint gateways.